main.js

```diff
@@ -1,10 +1,123 @@
+//Functions to run at start of webpage
+makeRows()
```

Comment:
You don't need to start off with with empty rows, I know that's how it looks like in the wireframe but the table body can be empty since our data is dynamically created.